app/src/main/res/layout/fragment_word_search_help.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/vocabulary_filter"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingTop="250dp"
+    android:paddingBottom="250dp"
+    android:paddingLeft="10dp"
+    android:paddingRight="10dp"
+    android:background="@drawable/word_search_dialog_background_shape">
+
+    <TextView
+        android:id="@+id/tvTitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentTop="true"
+        android:layout_marginTop="10dp"
+        android:layout_marginHorizontal="20dp"
+        android:textColor="@color/black"
+        android:textSize="22sp"
+        android:textStyle="bold"
+        android:text="@string/word_search_puzzle" />
+
+    <TextView
+        android:id="@+id/tvHint"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/tvTitle"
+        android:layout_marginTop="10dp"
+        android:layout_marginHorizontal="20dp"
+        android:textColor="@color/black"
+        android:textSize="18sp"
+        tools:text="Look for the 🇪🇸 Spanish words in the word search grid. Tap on an English word to see its translation!" />
```

Comment:
Is this meant to be hardcoded to the language? ^

Reply:
The tools text is hardcoded in the XML file for display purposes while I was setting up the layout, but the actual text is set at runtime with the appropriate target language [here](https://github.com/nadintamer/Lexis/pull/13/files#diff-4df13795d9f3b866d198bb0e61c4774104f6e9bd9501e172b6d8eb19f3798649R64)!